src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -72,6 +118,18 @@
     req
     (restrict-query ["=" ["node" "active"] true] req)))
 
+(defn restrict-query-to-active-nodes'
```

Comment:
I see you've also added :deprecated metadata to the original version; that's convincing me a bit more that this convention is good.